src/modules/interface/powertoy_system_menu.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+
+class PowertoyModuleIface;
+
+class PowertoySystemMenuIface {
+public:
+  struct ItemInfo {
+    std::wstring name{};
+    bool enable{ false };
+    bool checkBox{ false };
+  };
+  /* 
+   * Set configuration of system menu items for specific powertoy module. Configuration
+   * parameters include item name (and hotkey), item status at creation (enabled/disabled)
+   * and whether check box will appear next to item name when action is taken.
+   */
+  virtual void SetConfiguration(PowertoyModuleIface* module, std::vector<ItemInfo>& config) = 0;
```

Comment:
Looks like `config` should be a const ref, it does not get changed in SetConfiguration